### PR TITLE
render: initialize klog flag

### PR DESCRIPTION
**Pull request by Tal-or**
klog flags are not initiated for the render command, so
passing klog arguments like `--v` would be ignored

In addition, added a few logs for debugging purposes.

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: hack/render-sync.sh [hunk 1]
```diff
@@ -5,9 +5,8 @@ ARTIFACT_DIR=$(mktemp -d)
...
 _output/cluster-node-tuning-operator render \
---performance-profile-input-files "${WORKDIR}"/test/e2e/performanceprofile/cluster-setup/manual-cluster/performance/performance_profile.yaml \
---asset-input-dir "${WORKDIR}"/build/assets \
+--asset-input-dir "${WORKDIR}"/test/e2e/performanceprofile/cluster-setup/manual-cluster/performance,"${WORKDIR}"/test/e2e/performanceprofile/cluster-setup/base/performance \
 --asset-output-dir "${ARTIFACT_DIR}"
 
-cp -r "${ARTIFACT_DIR}"/*  "${WORKDIR}"/test/e2e/performanceprofile/testdata/render-expected-output/
-rm -r "${ARTIFACT_DIR}"
+cp "${ARTIFACT_DIR}"/*  "${WORKDIR}"/test/e2e/performanceprofile/testdata/render-expected-output
+rm -rf "${ARTIFACT_DIR}"
```
Comment (yanirq) [inline hack/render-sync.sh:12]: worth checking that the dir is populated with something right or at least have a default option. this is a bit risky

File: pkg/performanceprofile/cmd/render/render.go [hunk 2]
```diff
@@ -66,6 +66,7 @@ func render(inputDir, outputDir string) error {
 
 	// Read asset directory fileInfo
 	filePaths, err := listFiles(inputDir)
+	klog.V(4).Infof("listed files: %v", filePaths)
```
Comment (yanirq) [inline pkg/performanceprofile/cmd/render/render.go:69]: why V(4) ?

Comment (Tal-or) [reply pkg/performanceprofile/cmd/render/render.go:69]: this is for debugging purposes. I don't see the need for that to always be log

File: hack/render-sync.sh (hunk 1 above)
Comment (Tal-or) [reply hack/render-sync.sh:12]: I'll remove the `-f` but this directory is the artifact of  `mktemp` so I don't think there is something to worry about

File: pkg/performanceprofile/cmd/render/cmd.go [hunk 3]
```diff
@@ -55,7 +56,7 @@ func NewRenderCommand() *cobra.Command {
 	}
 
 	renderOpts.AddFlags(cmd.Flags())
-
+	addKlogFlags(cmd)
```
Comment (fromanirh) [inline pkg/performanceprofile/cmd/render/cmd.go:59]: (semi-)nit: klog flags, or in general flags pertaining to utility modules, are usually added first, then the application-specific flags.

**Review by fromanirh — COMMENTED** [review]
/approve
/lgtm

File: pkg/performanceprofile/cmd/render/cmd.go (hunk 3 above)
Comment (Tal-or) [reply pkg/performanceprofile/cmd/render/cmd.go:58]: Done. Thanks

**Review by ffromani — COMMENTED** [review]
/approve
/lgtm

**Comment by ffromani** [comment]
/retest
infra issues (and known gcp-pao lane flakiness)

**Pull request merged**